### ISSUE TITLE
The function of  WarmUpLR() in the utils.py?

### ISSUE THREAD

**Issue by HTL2018** [weiaicunzai/pytorch-cifar100#10]
thank for your great code.
I am confused of the function of  WarmUpLR() in the utils.py? And the lr_finder.py?

**Comment by weiaicunzai** [comment]
At the  beginning of training phase, we could slowly increase the learning rate from 0 to the given learning rate, to stabilize the training phase.This trick was introduced in ResNet paper, yolo also used this training trick.
